src/js/ia/IAPage.js

```diff
@@ -1522,6 +1527,19 @@
                         });
                     }
 
+                    // Called on userpages links click
+                    function sendReq($obj) {
+                        var username = $.trim($obj.text());
+                        var id = $obj.attr("id").match(/maintainer/)? "maintainer" : "contributor";
+                        var url_suffix = "?" + Math.ceil(Math.random() * 1e7);
+                        var img = new Image();
+
+                        username = username? "_" + username : "";
+
+                        $obj.append(img);
```

Comment:
@bsstoner Oh, I added that because it didn't seem to work without it, previously, but I just tried commenting that out and looks like it wasn't necessary! 
I was probably doing something else wrong, then :smile: 

Just updated the code!

Thanks!